packages/grafana-data/src/types/logs.ts

```diff
@@ -60,6 +60,7 @@ export interface LogRowModel {
   searchWords?: string[];
   timeFromNow: string;
   timeEpochMs: number;
+  timeEpochNs: number;
```

Comment:
We cannot use number for ns because it overflows safe javascript number

Reply:
Also would be nice to add a comment why we need this in addition to timeEpochMs

Reply:
@aocenas changed to strings and adjusted sort methods